### ISSUE TITLE
x.btn background color

### ISSUE THREAD

**Issue by Drauku** [Weilbyte/PVEDiscordDark#22]
Found another background color issue.
![image](https://user-images.githubusercontent.com/17952866/86516176-03abb800-bde4-11ea-9904-e10cf9fa7cbe.png)

I tried to add the code for this myself, but I don't think I know enough about CSS/HTML to do so. I forked and updated the CSS, then tried to test the fix on my own installation, but I could not get the background color to change.

My attempt inserted the below at line 275:
```css
.x-btn.x-unselectable.x-box-item.x-toolbar-item.x-btn-default-toolbar-small.x-btn-focus {
background-color:#401314;
}

```
The `x-btn-focus` class at the end is only visible when the button is focused, which only seems to occur after a dialogue window, produced by clicking the button, disappears.
I used the same background color as the recently fixed `proxmox-invalid-row`.

**Comment by Weilbyte** [comment]
Hi! 

Try using this: 
`.x-btn-focus.x-btn-default-toolbar-small {
    background-color: #401314;
}`

**Comment by Weilbyte** [comment]
Pushed the fix (dc9ce60)!
